go.mod

```diff
@@ -115,6 +115,7 @@ require (
 	github.com/hashicorp/terraform-plugin-go v0.14.3 // indirect
 	github.com/hashicorp/terraform-plugin-log v0.7.0 // indirect
 	github.com/hashicorp/terraform-plugin-mux v0.8.0 // indirect
+	github.com/hashicorp/terraform-provider-google-beta v1.20.1-0.20230322214054-60416c59ce67 // indirect
```

Comment:
This should be a direct dependency, and the non-beta provider shouldn't be a dependency any more.

Reply:
Okay thanks Stephen for the direction!
Let me have a look